### ISSUE TITLE
Placeholder not getting announced on Chrome

### ISSUE THREAD

**Issue by svenka22** [nvaccess/nvda#9354]
### Steps to reproduce:

- Go to https://login.microsoftonline.com on Chrome (this does not happen on Edge or other browsers)
- Have NVDA running
- Navigate to the username text box 
- Find attached [NVDA not reading placeholder  after selecting use another account.zip](https://github.com/nvaccess/nvda/files/2947218/NVDA.not.reading.placeholder.after.selecting.use.another.account.zip)

### Actual behavior:

- Notice NVDA doesn't announce placeholder text

### Expected behavior:

- NVDA should announce the placeholder text

#### NVDA installed/portable/running from source:

- NVDA installed

#### NVDA version:

- 2018.4.1

#### Chrome version:

Version 72.0.3626.121 (Official Build) (64-bit)

#### Windows version:

- Windows 10 Enterprise

#### Does the issue still occur after restarting your PC?

- Yes

#### Have you tried any other versions of NVDA?

- No

**Comment by JulienCochuyt** [comment]
I cannot reproduce on the mentioned web page with latest Chrome 75.0.3770.100
Placeholder successfully reported on my side by NVDA 2018.4.1 portable and 2019.2beta2 installed.
An update of the web site or the updated Chrome might have solved this.

@svenka22, do you still encounter this issue?

**Comment by Adriani90** [comment]
@svenka22 I am testing on NVDA 2019.3.1 and Crome 81, I cannot reproduce this issue either. I am closing this for now as works for me.
But please don't hesitate to comment on it if you are still reproducing this issue, then we can reopen it.
